src/Helper/Api/ApiService.php

```diff
@@ -110,23 +111,7 @@ private function createContentFileHashField(string $apiName, UploadedFile $file)
         ];
     }
 
-    public function addClientRequest(ClientRequest $clientRequest)
-    {
-        $name = $clientRequest->getOption('[api][name]', false);
-
-        if ($name) {
-            $this->clientRequests[$name] = $clientRequest;
-        }
-    }
```

Comment:
It's used the core bundle. Please run the phpstan in the core as well

Reply:
Check the full coreBundle but addClientRequest is not called, this is on hte apiService helper. The function was in the first version of the ApiService, after the clientRequests were injected in the controller and we forgot to remove this function.

Reply:
The core only uses the Environment and EnvironmentHelper class, plus it has custom route for ems_core_asset_proxy which goes to EMS\ClientHelperBundle\Controller\AssetController:proxyFromRefererPattern. 

In a next version/pr I will implement contract interfaces in the clientHelperBundle and the core and formBundle will only use interfaces of the clientHelper. That way we see when we break something